threedi_api_client/api.py

```diff
@@ -149,7 +159,7 @@ def __init__(
         client_secret = user_config.get("THREEDI_API_CLIENT_SECRET")
 
         basic = all(x for x in (username, password))
-        tokens = all(x for x in (access_token, refresh_token))
+        tokens = bool(access_token)
```

Comment:
Make one of the above mentioned renewal options mandatory in case access_token is supplied?

Reply:
I don't think it is necessary; you could construct an (temporary) ApiClient with a temporary token (I do that a lot actually)